Functions/Gherkin.ps1

```diff
@@ -576,72 +628,74 @@ function Invoke-GherkinScenario {
     Exit-MockScope
 }
 
-function Find-GherkinStep {
```

Comment:
I meant to completely remove this--not just comment it out. I guess I should first solicit comments on the removal of this unused function--because perhaps it's there for a reason? Depending on the responses I get, I'll either drop this commit or rebase the _actual_ removal of the function.

Reply:
This is meant for tooling --hypothetically-- your editor can call this when it sees you writing in a .feature file to show you intellisense, to auto-complete "when ..." sentences, or highlight _missing_ steps, etc.  It's basically the first step toward integrating with the PSES...

Reply:
OH!! Definitely useful in the future. Ok, I’ll drop that commit. Thanks!!

Reply:
Wow, I recently used `Find-GherkinStep` to find an existing step and execute it from within another step to help keep my test code dry. So this is definitely a useful cmdlet.